docs/content/AnchoredOverlay.mdx

```diff
@@ -81,6 +81,11 @@ See also [Overlay positioning](/Overlay#positioning).
       </>
     }
   />
+  <PropsTableRow
+    name="anchorId"
+    type="string"
+    description="Optional: An override to the internal id that will be passed to the anchor."
```

Comment:
Don't think you'll need to add optional here, as it's assumed they are all optional unless flagged as required.